### PR TITLE
Feat/profile_v3

### LINKED ISSUE #9873: Profile v3 - main UI
going offchain - partially 

# Design 
image
- updated in Figma, page profile v3 : https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=2116-13786&mode=design&t=3ECZpbp3kGl9oWOO-11
-  main difference is the top part, **bottom one (nft previews) stays the same** 
- extended with profile picture, banner, follow function, bio, different stats 
- **Introduced 14px** font size—finally, the time came to use this size—sometimes 16px is just too big, and 12px is just too small. 
- **look for font sizes** - for example in the stats section, different sizes than we are used to are used 

letsgo

## Banner and profile picture 
- there is banner of size **1440x360** - i guess 360px will be the maximum height
- on top of the banner, there is 144x144px big profile picture (padding is 10px - the actually img is only 124px big) 

the rest is underneath 

## Top part
- lets start with artist name - this will be the offchain name, if the offchain name is not set, it will show the address

under are buttons
- **follow** - states of the follow button are in the component section
- **wallet and links** - new one, it consists of wallet address and links the user predefined 
image
   - copy button, transfer (redirect to our transfer page with prefilled address, subscan, subID) 
- **share button** 
image

## Bio
- limited to **200 characters** (in the edit profile modal - issue will be linked in the tracking issue) 
- 14px font size 

## Followed by 
- this will show people that follow the account, and you follow them - same as Twitter, Instagram, etc. 

## Stats 
image
- bigger this time, bigger font size, different data

then divider and the rest stays the same, now for different states 

# Different views

## Visitor 
- visitor will see **follow button and the things you defined in the profile creation** 
this is how empty profile looks like (user did not created the offchain profile yet) 
image
- placeholder banner, placeholder profile picture, no bio 

## Owner 
this is how the owner will see the empty account 
image
- instead of the follow function, there is **"create profile"** 

if the profile is already created, **there is edit profile** 
image

# Mobile
image

stacks up the components

# Dark mode
- following the **rules of our color system and color pairs** 
image

# Components section
all useful **components** found here 
image

issue 1/3

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Feature

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #9873

## Extra Info

design wise i think it's good to go
~~codewise still missing transaltions~~ **done**

and obviously there is no where to fetch profile info from yet

### useful url params

to see new design use `redesign=true`
to see mock data use `mock=true`

**Example**

https://deploy-preview-9931--koda-canary.netlify.app/ahp/u/1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8?redesign=true&mock=true 

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

image

### CODE REVIEW

**Comment by daiagi** [comment]
Mobile issue

**Comment by exezbcz** [comment]
looking good! really nice 



only thing i catched is the hover over these button
image
iirc it was changed recently 

here: 
- https://github.com/kodadot/nft-gallery/pull/9747

**Comment by daiagi** [comment]
@prury 

failing tests seem unrelated to this PR

**Comment by daiagi** [comment]
@exezbcz 

>only thing i catched is the hover over these button


fixed

**Comment by daiagi** [comment]
all ready

**Comment by yangwao** [comment]
Thanks @daiagi 
I think this is much more needed to continue on other things I guess?

**Comment by daiagi** [comment]
@yangwao could you rephrase please?

**Comment by daiagi** [comment]
@daiagi  could you rephrase please?

**Comment by vikiival** [comment]
> @yangwao could you rephrase please?

@yangwao I guess we need to implement at least MVP for profile worker - https://github.com/kodadot/private-workers/issues/103

This PR is basically a mock, so not usable for prod

**Comment by daiagi** [comment]
That's not accurate
The mock data is there so @exezbcz could review the design
It can be removed before merging

The pr can still got to prod. But it would not display any new user profile data until we have BE for it

And anyway it's hidden under redesign url param until we will be ready to switch

**Comment by yangwao** [comment]
looking https://deploy-preview-9931--koda-canary.netlify.app/ahp/u/15CoYMEnJhhWHvdEPXDuTBnZKXwrJzMQdcMwcHGsVx5kXYvW?redesign=true&mock=true

this could be bit smaller font and probably closer to line?
cc @exezbcz 


image

dropdown
I would put farcaster, x and web order
image

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 9.55 USD/DOT ~ 5.236 $DOT
🧗 [1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8](https://polkadot.subscan.io/account/1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)
🔗 [0x28848e3aebb51d63084ca0608053aedd91dd8bdf88db888e3f95ee535ff335c5](https://polkadot.subscan.io/extrinsic/0x28848e3aebb51d63084ca0608053aedd91dd8bdf88db888e3f95ee535ff335c5)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)
